recipes/mender-snapshot/debian-master/control

```diff
@@ -0,0 +1,14 @@
+Source: mender-snapshot
+Maintainer: Mender Team <mender@northern.tech>
+Section: admin
+Priority: optional
+Standards-Version: 3.9.2
+Build-Depends: debhelper (>= 9)
+
+Package: mender-snapshot
+Architecture: any
+Depends: ${misc:Depends}
+Description: Mender Snapshot is an standalone tool to create an snapshot of the
```

Comment:
Nit: `a standalone` I think

Reply:
nit-nit: a snapshot as well.
```suggestion
Description: Mender Snapshot is a standalone tool to create a snapshot of the
```

Reply:
Thank you. Submitted also here:
* https://github.com/mendersoftware/mender-snapshot/pull/4